crates/forge/src/main.rs

```diff
@@ -36,6 +36,24 @@ struct Args {
     /// Stop test execution after first failed test
     #[arg(short = 'x', long)]
     exit_first: bool,
+
+    /// Number of fuzzer runs
+    #[arg(short = 'r', long, value_parser = validate_runs)]
+    fuzzer_runs: Option<u32>,
+
+    /// Seed for the fuzzer
+    #[arg(short = 's', long)]
+    fuzzer_seed: Option<u64>,
+}
+
+fn validate_runs(val: &str) -> Result<u32> {
+    let parsed_val: u32 = val
+        .parse()
+        .map_err(|_| anyhow!("Failed to parse {val} as u32"))?;
```

Comment:
This will result in an overflow error instead of a more human-readable message when it exceeds u32 size. Although this might be not be a concern since users should not use values that large anyway.

Reply:
In case of the overflow we still get an error like
```shell
snforge --fuzzer-runs 99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999
error: invalid value '99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999' for '--fuzzer-runs <FUZZER_RUNS>': Failed to parse 99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999 as u32
```

Isn't it okay?